libs/log/default.go

```diff
@@ -29,7 +30,17 @@ func NewDefaultLogger(format, level string, trace bool) (Logger, error) {
 	var logWriter io.Writer
 	switch strings.ToLower(format) {
 	case LogFormatPlain, LogFormatText:
-		logWriter = zerolog.ConsoleWriter{Out: os.Stderr}
+		logWriter = zerolog.ConsoleWriter{
+			Out:        os.Stderr,
+			NoColor:    true,
+			TimeFormat: time.RFC3339,
+			FormatLevel: func(i interface{}) string {
+				if ll, ok := i.(string); ok {
+					return strings.ToUpper(ll)
+				}
+				return "????"
```

Comment:
Wait, why are we doing this?

Reply:
This is same semantics as the default behavior in zero log (use question marks for unknown/invalid levels).

Reply:
see here: https://github.com/rs/zerolog/blob/98f889fcded426be57acc4aab62d615a2d679f0b/console.go#L331